test/Sema/explicit_lifetime_dependence_specifiers.swift

```diff
@@ -6,6 +6,14 @@ struct Container {
   let ptr: UnsafeRawBufferPointer
 }
 
+struct AnotherBufferView : ~Escapable {
```

Comment:
I AnotherBufferView inferred to be BitwiseCopyable? Do you want to specify BitwiseCopyable here to be explicit?